patches/minecraft/net/minecraft/client/gui/components/DebugScreenOverlay.java.patch

```diff
@@ -79,10 +79,16 @@
 +   protected void drawGameInformation(GuiGraphics p_281525_) {
 +      List<String> list = this.getGameInformation();
 +      list.addAll(this.getOverlayHelp());
++      net.minecraftforge.client.ForgeHooksClient.onCustomizeDebugEvent(p_281525_, minecraft.getWindow(), minecraft.getFrameTime(), list, true);
 +      this.renderLines(p_281525_, list, true);
```

Comment:
Not sure why it made a change to all these lines. 
The decompiled source has the wrong indentation. When I tried to fix that, it made the patch even larger.

Reply:
Not sure if you're just reading the diff wrong, but the only change this PR makes is this line you've added on line 82. GitHub highlights all diff values in a patch file, so you have to look at the line numbers themselves to see what's actually changed.

Reply:
gotcha